neuralprophet/configure.py

```diff
@@ -96,6 +98,25 @@ class Train:
     loss_func_name: str = field(init=False)
 
     def __post_init__(self):
+        # convert to list
+        if not isinstance(self.quantiles, list):
```

Comment:
maybe we can just assert that it is a list, instead of wrapping it in a list, to avoid issues with None cases etc?

Reply:
Done